astro/src/content/blog/i-just-want-authentication-to-work.mdx

```diff
@@ -0,0 +1,51 @@
+---
+publish_date: 2023-10-19
+title: I Just Want Authentication To Work
+description: When I am writing an Application the last thing that I care about is setting up Authentication, I just want it to work!
+authors: Alex Patterson
+image: 
+categories: Company, News
+tags: award, security mag
+excerpt_separator: "{/* more */}"
+---
+
+Authentication is the process of verifying the identity of a user or entity. It is a fundamental security measure that is used in a wide variety of applications, including web applications, mobile apps, and APIs. The thing is I just don't care about figuring it out! I just want it to work!!
+
+{/* more */}
+
+There are many different ways to implement authentication, but the most common approach is to use a username and password. Yep, just like back in 1960 when [The Everly Brothers](https://youtu.be/KtY3L5tDN_E?si=BATB3BzJJFpWsHmB) had one of the top songs out there, this method is still most popular today. In my opinion, just like I have never heard this song before, I don't want to hear about passwords any more either!
+
+Another common authentication method is to use a single sign-on (SSO) solution. SSO solutions allow users to log in to multiple applications using a single set of credentials. This can be convenient for users, as they do not have to remember multiple usernames and passwords. This method came about around the same time Eminem hit the [billboard top 100](https://youtu.be/6mPCy9LMxTI?si=5A2I-N1neb_8rGOl) so at least this is in my wheelhouse.
+
+My point is that Authentication is an important security measure, but it can also be complex to implement. This is why we have been hanging on to these old methods so long! When you are creating a new application (unless it is one for Authentication) the last thing you care about is trying to roll your own. There are many different authentication methods to choose from, and each method has its own advantages and disadvantages. Additionally, authentication needs to be implemented in a secure way to prevent attackers from gaining unauthorized access to applications. When all I care about creating is the next [Napster](https://youtu.be/CKrdsGdLVQ8?si=bTp4DHKS4tlpQzfW) so everyone can share their 1990's music with me again!
```

Comment:
"...the last thing you care about is trying to roll your own. "  I think this should be more along the lines of 
"the last thing you want to do is spend precious development cycles trying to roll your own security when you can get it for [free](https://fusionauth.io/download)."  or something like that.